src/amalgam/gen/scalar.c

```diff
@@ -14558,6 +14558,655 @@ void xnn_f32_vsqr_ukernel__scalar_u4(
   }
 }
 
+void xnn_qd8_f16_qb4w_gemm_minmax_ukernel_1x8__scalar(
```

Comment:
IIRC this amalgam file only contains `PROD` kernels. And we shouldn't be adding these to the prod list just yet. Or may be never. IIRC we might have to for benchmarking or something but that can be done differently.

Reply:
These get automatically pulled in the amalgam because they are in the gemm-config. I could potentially remove the gemm-config changes from this PR and save them for the subgraph PR. What do you prefer?

Reply:
Regenerated after pulling out gemm-config, changes are gone.